theme/parts/tabbox.css

```diff
@@ -0,0 +1,8 @@
+/*Tabbox*/
+
+@namespace xul "http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul";
+
+/* Main tab content */
+.tabbrowser-tabbox {
+	border: 1px solid rgba(0,0,0,0.9);
```

Comment:
After checking if the file is imported and with the correct selector, check again if the variable for the border doesn't work, depending on the context you might need to use `--gnome-toolbar-border-color`.